apps/api/src/routers/user.py

```diff
@@ -59,19 +59,21 @@ async def logout() -> RedirectResponse:
     return response
 
 
-@router.get("/me", response_model=IdentityResponse)
+@router.get("/me")
 async def me(
     user: Annotated[Union[User, None], Depends(use_user_identity)]
-) -> dict[str, object]:
+) -> IdentityResponse:
     log.info(user)
     if not user:
         return dict()
```

Comment:
Issue: return type should be IdentityResponse

Reply:
Weird, mypy should have caught this, will need to troubleshoot the checks workflow.

Reply:
Was getting filtered out to only added lines, removed filter in 6c03c32.